packages/rollup-plugin/src/index.ts

```diff
@@ -0,0 +1,240 @@
+import {
+    Stylable,
+    StylableExports,
+    StylableMeta,
+    visitMetaCSSDependenciesBFS,
+    emitDiagnostics,
+    DiagnosticsMode,
+} from '@stylable/core';
+import {
+    getUrlDependencies,
+    sortModulesByDepth,
+    calcDepth,
+    CalcDepthContext,
+    hasImportedSideEffects,
+} from '@stylable/build-tools';
+import { resolveNamespace as resolveNamespaceNode } from '@stylable/node';
+import { StylableOptimizer } from '@stylable/optimizer';
+import fs from 'fs';
+import { basename, extname, join, parse } from 'path';
+import { createHash } from 'crypto';
+import { Plugin, PluginContext } from 'rollup';
+import { getType } from 'mime';
+
+const production = !process.env.ROLLUP_WATCH;
+
+export interface StylableRollupPluginOptions {
+    optimization?: {
+        minify?: boolean;
+    };
+    inlineAssets?: boolean | ((filepath: string, buffer: Buffer) => boolean);
+    fileName?: string;
+    diagnosticsMode?: DiagnosticsMode;
+    resolveNamespace?: typeof resolveNamespaceNode;
+}
+
+const ST_CSS = '.st.css';
```

Comment:
we don't have any other package that reuses this value it's inline in all of them. it will probably will be refactored when we move to support `.stcss`